### DISCUSSION TITLE
Invitation for new member failed

### DISCUSSION THREAD

**Discussion by FlashGordonHH** [dani-garcia/vaultwarden#4952] [Q&A]
As the owner, I cannot invite new members. The system says that the user has been invited and everything is listed in the dashboard, but the new member never receives an invitation. Not even in the spam folder. What do I have to do?

**Comment by stefan0xC** [comment]
You need to [configure SMTP](https://github.com/dani-garcia/vaultwarden/wiki/SMTP-Configuration) for emails to be sent. Otherwise they should be able to just register.
